scripts/agent.yml

```diff
@@ -219,13 +219,14 @@ buildvariants:
       - localhost
     tasks:
       - name: my_group
-  - name: container_bv
-    display_name: container_bv
+  - name: pod_bv
+    display_name: pod_bv
     run_on:
       - container
     tasks:
       - name: container_task
 
+# TODO: Use more official image repo such as an AWS ECR image repo
```

Comment:
Is there a ticket number for this?

Also, based on the way the smoke test is set up, I don't think the particular image even matters here since the configuration does not affect the smoke test. Maybe we can just plug in a dummy value (e.g. `not_a_real_image`) and add a comment above the `containers` section saying that the container configuration specified is irrelevant since it's not actually used in the smoke test to start a container?

Reply:
You're right it doesn't currently matter - I just included it in anticipation of future improvements to this test that will need the repotracker.  I'll change the image name and update the comment saying it's not used for now.

Reply:
I'll include the work of the ticket with what you mentioned [here](https://github.com/evergreen-ci/evergreen/pull/5883#discussion_r965312592).

Ticket created here and included in comment: https://jira.mongodb.org/browse/EVG-17658